scripts/docker-as-drupal

```diff
@@ -898,6 +910,117 @@ elif [ "$1" = "behat" ]; then
   # exit
   exit $behat_exit
 
+#
+# RUN NIGHTWATCH TESTS
+#
+elif [ "$1" = "nightwatch" ]; then
+  shift
+
+  SKIP_DEPENDENCIES=0
+  SKIP_STYLEGUIDE=0
+  SKIP_DB_RESET=0
+  SKIP_DEFAULT_CONTENT=0
+  NIGHTWATCH_GROUPS=()
+  NIGHTWATCH_ARGV=()
+
+  while [ $# -gt 0 ]; do
+    case "$1" in
+      --skip-dependencies)
```

Comment:
I prefer `--with-dependencies`, the case where we don't have already installed it is not comon

Reply:
I use the same pattern you use for bootstrap

Reply:
Locally, bootstrap command is used to install dependencies, on codeship they are installed on build. Most time you don't have to update them when you run tests (never have to do on codeship), so it's why in tests commands I pefer `--with-dependencies` but have `--skip-dependencies` in bootstrap command.

Reply:
not the Nightwatch dependencies. They should be installed in web/core which are not installed in our boostrap.